installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc

```diff
@@ -12,6 +12,14 @@ toc::[]
 * You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
 * You have read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
 
+include::modules/virt-planning-bare-metal-cluster-for-ocp-virt.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
Include the `[role="_additional-resources"]` declaration before the heading. Applies to the IPI Prerequisites assembly as well.

Reply:
Thanks for catching that! Somehow I thought that only applied to `==` add'l resources... 🤦🏻